src/selectedtests/work_items/process_work_items.py

```diff
@@ -125,6 +131,8 @@ def _run_create_task_mappings(
         mongo.task_mappings().insert_many(transformed_mappings)
     log.info("Finished task mapping work item processing")
 
+    return True
```

Comment:
This function is supposed to return a `bool` and callers trigger off that to know if a work item can be completed.

Reply:
Whoops, nice catch! 

Would you mind adding a `return True` to `_run_create_test_mappings` as well, so the bug is fixed there as well?

Reply:
done